src/get_context_sequences.py

```diff
@@ -0,0 +1,215 @@
+from lxml import etree
+import argparse
+from pyriksdagen.utils import elem_iter, protocol_iterators
+import re
+import progressbar
+import pandas as pd
+
+####    utility functions   ####
+def cleanup_segments(text_seq):
```

Comment:
Using .split() without an argument should split on all whitespace, no need to do the translate part.